src/routes/trainerMatchingRoute.js

```diff
@@ -0,0 +1,25 @@
+const express = require("express");
+
+const { trainerMatchingController } = require("../controllers");
+const { asyncWrap } = require("../middlewares/errorHandler");
+
+const trainerMatchingRouter = express.Router();
+
+trainerMatchingRouter.get(
+  "/",
+  asyncWrap(trainerMatchingController.getTrainerProduct)
+);
+trainerMatchingRouter.post(
+  "/",
```

Comment:
```suggestion
trainerMatchingRouter.get(
  "/products",
  asyncWrap(trainerMatchingController.getTrainerProduct)
);
trainerMatchingRouter.post(
  "/product",
```

1. [GET] "/trainer" 는 트레이너의 정보를 받아오는 API인 것 같은 의미를 띕니다.
2. 프론트와 통신상 조율이 가능하다면, uri 업데이트 하시는 것이 좋을 것 같습니다. (/trainer-proucts 혹은 /trainer/product 혹은 /training 을 추천드립니다)

Reply:
확인해서 training 으로 변경 했습니다!
/training 으로 해서 운동하는 것에 관련된 정보를 받아오는 걸로 보여지게 했어요!